### PR TITLE
Docs updates

### PR DESCRIPTION
**Pull request by craicoverflow**
Closes https://github.com/aerogear/graphback/issues/888

## Tasks

- [x] Add use case for graphql-serve
- [x] graphback.dev/docs/plugins/crud-resolvers Plugin mentions runtime. We should link to examples rather than putting this into plugin
- [x] graphback.dev/docs/db/datasources - is in wrong chapter
Database chapter can be renamed to Database migrations
- [x] Migrating to Graphback -> Migrating from REST
- [ ] Regenerate website

### CODE REVIEW

**Comment by craicoverflow** [comment]
> graphback.dev/docs/db/datasources - is in wrong chapter/ Database chapter can be renamed to Database migrations

@wtrocki I don't see anything wrong with datasources being in the Database chapter.

**Comment by craicoverflow** [comment]
@ssd71 I've added documentation for GraphQL Serve :+1:

File: docs/intro/existingproject.md [hunk 1]
```diff
@@ -21,18 +21,16 @@ Graphback will do the following:
...
+1. To change the default application configuration see [Graphback Config](./config.md).
 
-To change the default application configuration see [Graphback Config](./config.md).
+2. See [Defining your Data model](./datamodel.md) for how to design your data model.
 
-Once your data model is complete, run `graphback generate` from your project to generate a GraphQL schema and API.
+3. Graphback provides a number of different data sources which can be configured at application runtime. See [Data Sources](../db/datasources).
 
-You can migrate your database to match your schema by running `graphback db`. See [Database Migrations](../db/migrations.md) for more.
+4. You can migrate your database to match your schema by running `graphback db`. See [Database Migrations](../db/migrations.md) for more.*
```
Comment (wtrocki) [inline docs/intro/existingproject.md]: For relational databases...

File: docs/intro/quickstart.md [hunk 2]
```diff
@@ -26,12 +26,18 @@ The CLI will ask you to choose from one of two templates:
...
+2. See [Defining your Data model](./datamodel.md) for how to design your data model.
 
-Once your data model is complete, run `graphback generate` from your project to generate a GraphQL schema and API.
+3. Once your data model is complete, run `graphback generate` from your project to generate a GraphQL schema and API.
 
-You can migrate your database to match your schema by running `graphback db`. See [Database Migrations](../db/migrations.md) for more.
\ No newline at end of file
+4. Graphback provides a number of different data sources which can be configured at application runtime. See [Data Sources](../db/datasources).
+
+5. You can migrate your database to match your schema by running `graphback db`. See [Database Migrations](../db/migrations.md) for more.*
```
Comment (wtrocki) [inline docs/intro/quickstart.md]: Same as above

**Pull request merged**